CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 - #2206 fix sonar warnings; some package versions had to be increased
 - #2213 - Show/Hide Dialog Field TouchUI Widget: Fix hidden required field not disabled to save the dialog 
 - Fixed JcrJsonAdapter IllegalStateException when writing multi-valued JCR properties
+- #2199 Add read permission for acs-commons-email-service user in conf folder using rep policy
```

Comment:
This should now be up in the `## Unreleased` section

Reply:
@HitmanInWis : Moved it to unreleased.